WordPress/Classes/ViewRelated/Comments/Content/CommentDetailContentTableViewCell.swift

```diff
@@ -0,0 +1,98 @@
+import UIKit
+import SwiftUI
+
+final class CommentDetailContentTableViewCell: UITableViewCell, Reusable {
+
+    // MARK: - Typealias
+
+    typealias ContentConfiguration = CommentDetailContentView.Configuration
+    typealias HeaderConfiguration = CommentContentHeaderView.Configuration
+    typealias MenuConfiguration = CommentContentHeaderView.MenuConfiguration
+    typealias MenuOption = CommentContentHeaderView.MenuConfiguration.Option
+
+    // MARK: - Views
+
+    private let stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.spacing = CGFloat.DS.Padding.single
+        stackView.alignment = .fill
+        return stackView
+    }()
+
+    private var authorHostingController: UIHostingController<CommentContentHeaderView>?
+
+    private var commentView = CommentDetailContentView()
+
+    // MARK: - Init
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        self.setup()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setup() {
+        self.stackView.addArrangedSubview(commentView)
+        self.stackView.translatesAutoresizingMaskIntoConstraints = false
+        self.contentView.addSubview(stackView)
+        self.contentView.pinSubviewToAllEdgeMargins(stackView)
+        self.selectionStyle = .none
+    }
+
+    // MARK: - Updating UI
+
+    func configure(with headerConfig: HeaderConfiguration, contentConfig: ContentConfiguration, parent: UIViewController) {
+        self.updateHeader(with: headerConfig, parent: parent)
+        self.commentView.configure(with: contentConfig)
+    }
```

Comment:
Just to double-check: what is the purpose of this initializer? It is not used and I'm not sure if we have a case for this.

Reply:
This acts as the primary `configure` method, while the other one is merely a convenience method. The convenience configure method is supposed to call this one, but I just realized that it's not the case. Resolved in 3d77d0a568c985c2bc3fb1515218c2d3df8cc9b5